docs/_includes/masthead.html

```diff
@@ -27,11 +27,13 @@
                     <form action="//search.warwick.ac.uk/website" role="search">
                         <input type="hidden" name="source" value="https://warwick.ac.uk/">
                         <div class="form-group">
-                            <label class="sr-only" for="id7-search-box">Search</label>
-                            <div class="id7-search-box-container">
+                            <div class="id7-search-box-container floating-label">
                                 <input type="search" class="form-control input-lg" id="id7-search-box" name="q" placeholder="Search Warwick" data-suggest="go" aria-label="Search Warwick">
+                                <label for="id7-search-box">Search Warwick</label>
```

Comment:
I think I'm right in thinking that if someone doesn't change the template, they just won't get the floating label, so this is a nice progressive enhancement

Reply:
>I think I'm right in thinking that if someone doesn't change the template, they just won't get the floating label, so this is a nice progressive enhancement

That's correct, yeah. Happy to do another JS DOM patching approach if we want it to apply everywhere from the get go, but I think this is probably fine as opt-in.

Will get the template updated.

Reply:
>I guess it's because of the ~ CSS selector, so this is fine.

Just for the record and my benefit, the reason this doesn't work is that we need to be able to select the label where it is followed by the input in the `:placeholder-shown` state. There is no sibling selector that works backwards, therefore we have to just use the slightly illogical DOM ordering that puts the label afterwards. This is fine though because we have the `for` attribute.